flake.nix

```diff
@@ -0,0 +1,34 @@
+{
+  description = "instantNIX";
+
+  nixConfig = {
+    extra-experimental-features = "nix-command flakes";
+
+    extra-substituters = [
+      "https://instantos.cachix.org"
+    ];
+
+    extra-trusted-public-keys = [
+      "instantos.cachix.org-1:6GezJZuxIK59ydzdyoIw6Yr4H6DaCoLzqhPaDhB8v24="
+    ];
+  };
+
+  outputs = {
+    self,
+    nixpkgs,
+  }: let
+    systems = [
+      "x86_64-linux"
+      "i686-linux"
+      "aarch64-linux"
+      "armv6l-linux"
+      "armv7l-linux"
+    ];
+    forAllSystems = f: nixpkgs.lib.genAttrs systems (system: f system);
+  in {
+    packages = forAllSystems (system:
+      import ./default.nix {
+        pkgs = import nixpkgs {inherit system;};
```

Comment:
```suggestion
        pkgs = nixpkgs.legacyPackages.${system};
```

Reply:
https://github.com/nix-community/nur-packages-template/blob/master/flake.nix#L18
also a common pattern in https://github.com/NixOS/templates

Reply:
It seems that using the `legacyPackages` output attribute instead of importing nixpkgs helps decrease the number of re-evaluation.

https://discourse.nixos.org/t/1000-instances-of-nixpkgs/17347/12